package.json

```diff
@@ -98,5 +98,8 @@
   },
   "lint-staged": {
     "*.{js,ts,vue}": "eslint --fix"
+  },
+  "dependencies": {
```

Comment:
Avoid dependencies in root package json - they will not be included in the released packages